tests/extreme_exos/show_fdb/extreme_exos_show_fdb.yml

```diff
@@ -0,0 +1,37 @@
+---
+parsed_sample:
+  - age: "0041"
+    interface: "1:2"
+    mac_address: "00:0c:29:4b:34:cf"
+    vlan_id: "0101"
+    vlan_name: "v101"
+  - age: "0041"
```

Comment:
do you want to capture leading `0`s for vlan_id and age?

Reply:
Hadn't really considered it, I'm fine with capturing them though. If you'd suggest otherwise also fine with changing it.

Reply:
I would rather not have leading 0s